### PR TITLE
Update front page content

### PR DESCRIPTION
**Pull request by achoum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,58 +31,51 @@ Temporian is available on PyPI. To install it, run:
 pip install temporian
 ```
 
-## Getting Started
+## Minimal example
 
-This is how a minimal end-to-end example looks like:
+Assuming the following dataset.
```
Comment (rstz) [inline README.md:36]: Nit: Assuming --> Consider

**Review by DonBraulio — APPROVED** [review]
lgtm 👍👍

File: README.md [hunk 2]
```diff
@@ -6,7 +6,22 @@
...
+
+## Key features
+
+- Temporian operates natively on **multivariate** and **multi-index time-series** and **time-sequences** data. With Temporian, all temporal data processing is unified.
+
+- Temporian favors **iterative** and **interactive** development in Colab, where users can **easily visualize intermediate results** 📊 each step of the way.
+
+- Temporian introduces a novel mechanism to **prevent unwanted future leakage** and **training/serving skew** 😰. Temporian programs always return the same result in batch and in streaming mode.
+
+- Temporian programs can run seemingly **in-process** in Python, on **large datasets using Beam** ☁️, and in **streaming for continuous** data ingestion.
```
[the remark this answers is not in the document]
Comment (achoum) [reply README.md:22]: *Done :)

File: README.md [hunk 3]
```diff
@@ -6,7 +6,22 @@
...
+
+- Temporian's core is implemented **in C++** and **highly optimized** 🔥, so large amounts of data can be handled in-process. In some cases, Temporian can provides speed-up in the order of 1000x compared to other libraries.
```
[the remark this answers is not in the document]
Comment (achoum) [reply README.md:24]: *Done

File: README.md (hunk 1 above)
Comment (achoum) [reply README.md:36]: *Done

File: README.md [hunk 4]
```diff
@@ -16,58 +31,51 @@ Temporian is available on PyPI. To install it, run:
...
 ```
 
-## Getting Started
+## Minimal example
 
-This is how a minimal end-to-end example looks like:
+Assuming the following dataset.
+
+```shell
+>>> head sales.csv
```
[the remark this answers is not in the document]
Comment (achoum) [reply README.md:39]: Nice.
*Done

File: README.md [hunk 5]
```diff
@@ -16,58 +31,51 @@ Temporian is available on PyPI. To install it, run:
...
+timestamps,store,price,count
+2022-01-01 00:00:00+00:00,CA,27.42,61.9
+2022-01-01 00:00:00+00:00,TX,98.55,18.02
+2022-01-02 00:00:00+00:00,CA,32.74,14.93
+2022-01-02 00:00:00+00:00,TX,48.69,83.99
+...
+```
+
+We compute the weekly sales per store as follow.
```
[the remark this answers is not in the document]
Comment (achoum) [reply README.md:48]: *Done

**Comment by achoum** [comment]
Thanks all.

**Pull request merged**